<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QFrame" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>796</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>800</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="main_page_logo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Main Page LOGO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Music Link :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="music_link_input">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>301</width>
      <height>50</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>paste music link here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>520</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Search for details</string>
    </property>
   </widget>
   <widget class="QLabel" name="main_page_developer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>689</y>
      <width>601</width>
      <height>111</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>âœ” App developed by Matin Ghanbari</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>main_page_logo</zorder>
   <zorder>label</zorder>
   <zorder>music_link_input</zorder>
   <zorder>search_button</zorder>
   <zorder>main_page_developer</zorder>
   <zorder>songWidget</zorder>
   <zorder>songWidget</zorder>
   <zorder>songWidget</zorder>
  </widget>
  <widget class="QWidget" name="songWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>800</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>740</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
    background-color: rgba(187, 0, 100, 0.68);
    border-radius: 12px;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="game_logo">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>45</y>
      <width>190</width>
      <height>190</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="qlabel_song">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>390</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;</string>
    </property>
    <property name="text">
     <string>Track Name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="trackName">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>390</y>
      <width>275</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>Track Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="qlabel_song_2">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>440</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;</string>
    </property>
    <property name="text">
     <string>Artist :</string>
    </property>
   </widget>
   <widget class="QLabel" name="qlabel_song_3">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>490</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;</string>
    </property>
    <property name="text">
     <string>Track Number :</string>
    </property>
   </widget>
   <widget class="QLabel" name="trackNumber">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>490</y>
      <width>275</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>trackNumber</string>
    </property>
   </widget>
   <widget class="QLabel" name="album">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>540</y>
      <width>275</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>album</string>
    </property>
   </widget>
   <widget class="QLabel" name="qlabel_song_4">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>540</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;</string>
    </property>
    <property name="text">
     <string>Album :</string>
    </property>
   </widget>
   <widget class="QLabel" name="qlabel_song_5">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>590</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;</string>
    </property>
    <property name="text">
     <string>Release Date :</string>
    </property>
   </widget>
   <widget class="QLabel" name="releaseDate">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>590</y>
      <width>275</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>releaseDate</string>
    </property>
   </widget>
   <widget class="QLabel" name="qlabel_song_6">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>640</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;</string>
    </property>
    <property name="text">
     <string>Duration (ms) :</string>
    </property>
   </widget>
   <widget class="QLabel" name="duration">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>640</y>
      <width>275</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>duration</string>
    </property>
   </widget>
   <widget class="QLabel" name="cover">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>75</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);border-radius:20px</string>
    </property>
    <property name="text">
     <string>cover</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="download">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>740</y>
      <width>341</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
    background-color: rgba(187, 173, 0, 0.68);
    border-radius: 12px;</string>
    </property>
    <property name="text">
     <string>Download Track</string>
    </property>
   </widget>
   <widget class="QLabel" name="textBack">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>531</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 136, 0, 0.44);
    border-radius: 25%;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="artist">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>440</y>
      <width>275</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:left;
padding:5px;
background-color:transparent;
padding: 5px;
font-size: 20px;
color:rgb(149, 250, 255);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <zorder>textBack</zorder>
   <zorder>back</zorder>
   <zorder>game_logo</zorder>
   <zorder>qlabel_song</zorder>
   <zorder>trackName</zorder>
   <zorder>qlabel_song_2</zorder>
   <zorder>qlabel_song_3</zorder>
   <zorder>trackNumber</zorder>
   <zorder>album</zorder>
   <zorder>qlabel_song_4</zorder>
   <zorder>qlabel_song_5</zorder>
   <zorder>releaseDate</zorder>
   <zorder>qlabel_song_6</zorder>
   <zorder>duration</zorder>
   <zorder>cover</zorder>
   <zorder>download</zorder>
   <zorder>artist</zorder>
  </widget>
  <widget class="QWidget" name="artistWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>800</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="backToSong">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>740</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
    background-color: rgba(187, 0, 100, 0.68);
    border-radius: 12px;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="game_logo_4">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>45</y>
      <width>190</width>
      <height>190</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="qlabel_song_19">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>390</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;</string>
    </property>
    <property name="text">
     <string>Artist Name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="artist_name">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>390</y>
      <width>275</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>Artist Name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="followers">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>440</y>
      <width>275</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>followers</string>
    </property>
   </widget>
   <widget class="QLabel" name="qlabel_song_20">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>440</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;</string>
    </property>
    <property name="text">
     <string>Followers :</string>
    </property>
   </widget>
   <widget class="QLabel" name="qlabel_song_21">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>490</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;</string>
    </property>
    <property name="text">
     <string>Genres :</string>
    </property>
   </widget>
   <widget class="QLabel" name="genres">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>490</y>
      <width>275</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>genres</string>
    </property>
   </widget>
   <widget class="QLabel" name="id">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>540</y>
      <width>275</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QLabel" name="qlabel_song_22">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>540</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;</string>
    </property>
    <property name="text">
     <string>ID :</string>
    </property>
   </widget>
   <widget class="QLabel" name="qlabel_song_23">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>590</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;</string>
    </property>
    <property name="text">
     <string>Popularity :</string>
    </property>
   </widget>
   <widget class="QLabel" name="popularity">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>590</y>
      <width>275</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>popularity</string>
    </property>
   </widget>
   <widget class="QLabel" name="qlabel_song_24">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>640</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;</string>
    </property>
    <property name="text">
     <string>Type :</string>
    </property>
   </widget>
   <widget class="QLabel" name="type">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>640</y>
      <width>275</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>type</string>
    </property>
   </widget>
   <widget class="QLabel" name="artist_pic">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>75</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);border-radius:20px</string>
    </property>
    <property name="text">
     <string>Artist Image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="spotifyProfile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>740</y>
      <width>341</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
    background-color: rgba(100, 173,200, 0.68);
    border-radius: 12px;</string>
    </property>
    <property name="text">
     <string>Spotify Profile</string>
    </property>
   </widget>
   <widget class="QLabel" name="textBack_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>531</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 136, 0, 0.44);
    border-radius: 25%;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>textBack_3</zorder>
   <zorder>backToSong</zorder>
   <zorder>game_logo_4</zorder>
   <zorder>qlabel_song_19</zorder>
   <zorder>artist_name</zorder>
   <zorder>followers</zorder>
   <zorder>qlabel_song_20</zorder>
   <zorder>qlabel_song_21</zorder>
   <zorder>genres</zorder>
   <zorder>id</zorder>
   <zorder>qlabel_song_22</zorder>
   <zorder>qlabel_song_23</zorder>
   <zorder>popularity</zorder>
   <zorder>qlabel_song_24</zorder>
   <zorder>type</zorder>
   <zorder>artist_pic</zorder>
   <zorder>spotifyProfile</zorder>
  </widget>
  <widget class="QWidget" name="downloadWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>800</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="download_logo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Main Page LOGO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>550</y>
      <width>400</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Downloading ...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="remain">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>600</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>remain time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <action name="actionDsafasf">
   <property name="text">
    <string>dsafasf</string>
   </property>
  </action>
  <zorder>downloadWidget</zorder>
  <zorder>mainWidget</zorder>
  <zorder>artistWidget</zorder>
  <zorder>songWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
